.gitignore

```diff
@@ -1,15 +1,9 @@
 *.jl.*.cov
 *.jl.cov
 *.jl.mem
-*.lp
-/Manifest.toml
-/test/Manifest.toml
-/docs/Manifest.toml
-/docs/build/
-env
-/.benchmarkci
-/benchmark/*.json
-coverage/
```

Comment:
These should remain, I've created an issue on Bestie for this.

Reply:
@abelsiqueira I put it back